ViewControllers/MovieDetail/ConstraintMovieDetailView.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  ConstraintMovieDetailView.swift
+//  Movies
+//
+//  Created by Gabriel Rodrigues Minucci on 29/06/2022.
+//
+
+import UIKit
+
+final class ConstraintMovieDetailView: UIView {
+    private(set) lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: 20, weight: .heavy)
+        label.textAlignment = .center
+        label.textColor = UIColor.white
+        label.numberOfLines = 0
+        return label
+    }()
+    
```

Comment:
Why have you added (set) to the access modifier? Why is it important here?

Reply:
It's not really necessary, but I like limiting access from other classes and try to only expose what is really necessary. Since we need to expose the view elements here (labels, collection, imageview, etc) I can not leave them as `private`, so be able to only get them I can set `private(set)` which means that setting is private but getting is public.
```swift
class MyClass {
    private(set) label = UILabel()
}

let a  = MyClass()
a.label.text = "text" // this is allowed
a.label = UILabe() // this won't be allowed
```